src/face.js

```diff
@@ -414,6 +415,7 @@ var face = function (key) {
             if (options) {
                 qs.name = options.name;
                 qs.userData = options.userData;
+				 qs.targetFace = options.targetFace;
```

Comment:
Hello,

Yes, of course. Sorry for the wrong ident.
On 1 Jun 2016 6:53 p.m., "Felix Rieseberg" <notifications@github.com> wrote:

> In src/face.js
> <https://github.com/felixrieseberg/project-oxford/pull/20#discussion_r65388897>
> :
>
> > @@ -414,6 +415,7 @@ var face = function (key) {
> >              if (options) {
> >                  qs.name = options.name;
> >                  qs.userData = options.userData;
> > +				 qs.targetFace = options.targetFace;
>
> Hey! Thanks for the PR - would you mind using the correct indentation here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/felixrieseberg/project-oxford/pull/20/files/94276a0b437e18f4a01d166a414fbd147d852a9d#r65388897>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFc6zKi_Y_nULV4Shzq9O3-pXQMO43J6ks5qHasGgaJpZM4IrKhn>
> .
>